pages/linux/xxd.md

```diff
@@ -0,0 +1,24 @@
+# xxd
+
+> Make a hexdump or do the reverse.
+> More information: <https://packages.debian.org/buster/xxd>.
+
+- Display Hex representation of a file:
+
+`xxd {{file}}`
+
+- Convert Hexdump into binary:
+
+`xxd -r {{file}}`
+
+- Produce Binary dump:
+
+`xxd -b {{file}}`
+
+- Print everything but the first three lines (hex  0x30  bytes):
+
+`xxd -s 0x30 {{file}}`
+
+- Print 120 bytes as continuous hexdump with 20 octets per line:
```

Comment:
I'm suggesting that because a quick search on "hexdump" would only return results with it separated as ["Hex Dump"](https://pt.wikipedia.org/wiki/Hex_dump).
```suggestion
- Print 120 bytes as continuous hex dump with 20 octets per line:
```

Reply:
How about we do:
- Print 120 bytes as continuous Hex dump with 20 octets per line:
instead of:
- Print 120 bytes as continuous hex dump with 20 octets per line:
Thanks for the suggestions :)